cmd/osctl/cmd/cluster.go

```diff
@@ -329,6 +333,7 @@ func saveConfig(input *generate.Input) (err error) {
 
 func init() {
 	clusterUpCmd.Flags().StringVar(&image, "image", "docker.io/autonomy/talos:"+version.Tag, "the image to use")
+	clusterUpCmd.Flags().StringVar(&networkMTU, "mtu", "1440", "MTU of the docker bridge network")
```

Comment:
We should probably default to `1500` like the docker daemon does.

Reply:
Updated.